flow-server/src/main/resources/plugins/theme-loader/theme-loader.js

```diff
@@ -3,7 +3,8 @@ const fs = require('fs');
 const path = require('path');
 
 // Collect groups [url(] [ |'|"]optional './|../', file part and end of url
```

Comment:
The comment should be updated as end of url is preceded with query.
Also on line 35 it should be noted that the groups have gone from 5 to 6
```suggestion
// Collect groups [url(] [ |'|"]optional './|../', file part, query and end of url
```

Reply:
Changed to have only 5 groups